lib/LedgerSMB/DBH.pm

```diff
@@ -38,11 +38,11 @@ sub connect {
     return undef unless $username && $password;
     return undef if $username eq 'logout';
 
-    my $dbh = DBI->connect(qq|dbi:Pg:dbname="$company"|, $username, $password,
+    my $dbh = DBI->connect("dbi:Pg:dbname=$company", $username, $password,
            { PrintError => 0, AutoCommit => 0,
              pg_enable_utf8 => 1, pg_server_prepare => 0 })
         or return undef;
-    $dbh->do("set client_min_messages = 'warning'");
+    $dbh->do('set client_min_messages = \'warning\'');
```

Comment:
Would be nice to use `q{}` here due to the escaping backslashes in the current solution.